.gitlab-ci.yml

```diff
@@ -0,0 +1,62 @@
+stages:
+  - deploy_to_sonatype
+  - create_key
+
+variables:
+  REGISTRY: 486234852809.dkr.ecr.us-east-1.amazonaws.com
+
+deploy_to_sonatype:
+  stage: deploy_to_sonatype
+  tags: [ "runner:docker", "size:large" ]
+  image: maven:3.8.2-jdk-8-slim
+  when: manual
+
+  script:
+    # Ensure we don't print commands being run to the logs during credential
+    # operations
+    - set +x
```

Comment:
Can we abstract this and make this a template (seems like a pattern we are/will be using)?

Reply:
I can copy the script I use and put it in the `README` of https://github.com/DataDog/agent-key-management-tools/tree/master/gpg for future plugins that will deploy to Sonatype.

Reply:
Not just a README, but a separate `sign.sh` script, so others can reuse and just fill in the blanks 🙂